templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+  <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">{{ user.username }}</a></li><br>
```

Comment:
Ensure that the `driver-detail` URL pattern expects a `pk` parameter corresponding to the user's ID. If the `Driver` model uses a different field for identification, adjust the URL accordingly.